ActiveDocuments/execCommand.html

```diff
@@ -759,23 +716,49 @@ <h2 id="methods-to-query-and-execute-commands">
         </li>
         <li>If the previous step returned false, return false.
         </li>
-        <li>If <var title="">command</var> is not in the <a href=
-        "#miscellaneous-commands">Miscellaneous commands</a> section, then
-        [=Dispatch=] an [= event=] at <var title="">affected editing host</var>
-        that uses the <a>EditingInputEvent</a> interface. The event's
-        {{Event/type}} attribute must be initialized to "input"; its
+        <li>If the action modified DOM tree, then [=Dispatch=] an [= event=] at
+        <var>affected editing host</var> using {{InputEvent}} interface. The
+        event's {{Event/type}} attribute must be initialized to "input"; its
         {{Event/isTrusted}} and {{Event/bubbles}} attributes must be
-        initialized to true; its <code title=
-        "dom-EditingInputEvent-command"><a href=
-        "#dom-editinginputevent-command">command</a></code> attribute must be
-        initialized to <var title="">command</var>; and its <code title=
-        "dom-EditingInputEvent-value"><a href=
-        "#dom-editinginputevent-value">value</a></code> attribute must be
-        initialized to <var title="">value</var>.
+        initialized to true; its {{InputEvent/inputType}} attribute must be
```

Comment:
NIT: DOM spec defines a nice hook algorithm "event construction steps"... you don't need to use them, but they might be useful here if you'd like. Maybe a followup :)

Reply:
Looking at [your work](https://w3c.github.io/payment-request/#merchantvalidationevent-constructor), it seems it's defined for each constructor. Since we are reusing an event interface from another spec, is it still applicable here?

Reply:
ah, you are right and great point. as it's just Event, it doesn't need any special set up.